src/python/other-bots/chromium-tests-syncer/run.py

```diff
@@ -51,6 +52,9 @@ def unpack_crash_testcases(crash_testcases_directory):
     if count >= MAX_TESTCASES:
       logs.info(f'{MAX_TESTCASES} testcases reached.')
       break
+    if count % TESTCASES_REPORT_INTERVAL == 0:
```

Comment:
Can also do it based on time: every minute, output progress.

Reply:
I'd leave that as a follow-up, in case we need it at all. Maybe a better query will reduce this iteration enough so that we don't need this.